UnleashedRecomp/main.cpp

```diff
@@ -180,12 +180,27 @@ void init()
 #endif
 }
 
+void SetWorkingDirectoryToExecutable()
+{
+    std::filesystem::path exePath;
+
+#ifdef _WIN32
+    exePath = std::filesystem::canonical(std::filesystem::path(_pgmptr));
```

Comment:
`_pgmptr` is currently deprecated and doesn't support Unicode characters. We should instead try to use the [`_get_wpgmptr`](https://learn.microsoft.com/en-us/cpp/c-runtime-library/reference/get-wpgmptr?view=msvc-170) function.

Reply:
Changed to GetModuleFileNameW instead for Windows as the usage of `_wpgmptr` (and the getter too) requires to process to run through either `wmain` or `wWinMain`.